migrations/sql/V2022.08.05.1000__AddGaviCountryEligibility.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE "gavi_support_eligibility_status" (
+  "id" INTEGER,
```

Comment:
You could set this as a `SERIAL PRIMARY KEY` type you will get the incrementing ID automatically. So wouldn't have to manually specify them lines 21 - 32. Though maybe you have deliberately not done that if this are IDs specified by gavi?

Reply:
Chatted about this, these are IDs set by gavi so we don't want an auto incrementing column